src/tsf/ConsoleTSF.cpp

```diff
@@ -32,7 +32,7 @@ HRESULT CConsoleTSF::Initialize()
     Init_CheckResult();
     _fCoInitialized = TRUE;
 
-    hr = _spITfThreadMgr.CoCreateInstance(CLSID_TF_ThreadMgr);
+    hr = ::CoCreateInstance(CLSID_TF_ThreadMgr, NULL, CLSCTX_ALL, IID_PPV_ARGS(&_spITfThreadMgr));

```

Comment:
Prefer nullptr over NULL where possible throughout these changes:
[ES.47: Use nullptr rather than 0 or NULL](http://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#es47-use-nullptr-rather-than-0-or-null)